misk/src/main/java/tdigest/tdigest.proto

```diff
@@ -0,0 +1,24 @@
+
+syntax = "proto3";
+
+package tdigest;
```

Comment:
Let's package this near the other protos:`option java_package = "com.squareup.digester.protos.tdigest";`